packages/fiori/cypress/specs/UploadCollection.cy.ts

```diff
@@ -0,0 +1,384 @@
+import { html } from "lit";
```

Comment:
Hello Siana,

lit-html is no longer the mount engine for the Cypress tests, we use preact + jsx (as for the component templates)

You will have to adopt this change in your PR. You can refer to this PR how to do this
https://github.com/SAP/ui5-webcomponents/pull/10653. Or, just check out the cypress tests in the main branch - they are all migrated already.